source/_posts/2016-10-14-backing-up-and-restoring-a-postresql-database.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "Backing Up and Restoring a PostgreSQL Database"
+date: 2016-10-14 14:05:38 +0200
+comments: true
+author: Jovana Dačić
+categories: 
+  - Ruby and Rails
+tags: 
+  - ruby
+  - rails
+  - PostgreSQL
+published: false
+---
+
+#Backing Up and Restoring a PostgreSQL Database
```

Comment:
@codalicious no need for a title here. It will be added automatically when page is generated.